server/.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  bracketSameLine: true,
+  singleQuote: true,
+  trailingComma: "all",
```

Comment:
```suggestion
  trailingComma: 'all',
```
Super small nitpick, just to match the `prettier` config from the `app/` folder.